src/main/java/org/opentripplanner/middleware/controllers/api/TripHistoryController.java

```diff
@@ -60,72 +61,56 @@ public void bind(final SparkSwagger restApi) {
                     .withName("userId")
                     .withRequired(true)
                     .withDescription("The OTP user for which to retrieve trip requests.").and()
-                .withQueryParam()
-                    .withName(LIMIT_PARAM_NAME)
-                    .withDefaultValue(String.valueOf(DEFAULT_LIMIT))
-                    .withDescription("If specified, the maximum number of trip requests to return, starting from the most recent.").and()
```

Comment:
Is sorting from most recents still happening?

Reply:
Yep, updated endpoint description.